languages.yaml

```diff
@@ -0,0 +1,2572 @@
+Ada:
+  type: programming
+  ascii: |
+    {0}                              *
+    {0}                              *
+    {0}                            * **
+    {0}   *                        *****
+    {0}   **                       ******
+    {0}    **                   ###********
+    {0}    ***          -******#'###*********
+    {0}     *****               #### *********
+    {0}      ****************************{1}XXX{0}**
+    {0}     ** ******************{1}XXXX{2}o{1}X{0}*{1}X{2}o{1}XXX{0}*
+    {0}      **** ***************{1}XXXX{2}ooooo{1}XX{0}*
+    {0}        *******************{1}XXXX{2}ooooo{1}X
+    {0}          *******************{1}XXX{2}ooo{1}X
+    {0}          ************************{1}XX{0}*
+    {0}             *************    ******
+    {0}  /| |                           ****
+    {0} /-|(|(|                     ********
+    {0}                        ************
+    {0}Time-tested, safe     *************
+    {0}and secure            ***********
+    {0}                         *****
+  colors:
+    ansi:
+      - default
```

Comment:
It would be best if we good find a way to use `white` and make it map to `default` in the template.